src/dma/collector/workflows/collection_extractor/base.py

```diff
@@ -32,24 +30,27 @@ async def execute(self) -> None:
         await super().execute()
         await self.extract_collection()
         await self.process_collection()
-        self.print_summary()
 
     async def extract_collection(self) -> None:
         collection = await self.collection_query_manager.execute_collection_queries()
         self.import_to_table(collection)
 
     async def process_collection(self) -> None:
```

Comment:
Do you intend to use this later?

Reply:
Yeah, intend to add in the "export files to zip" method here.  For now, it's basically a no-op by default.